### PR TITLE
Add SG Mix and Mega Draft League Version

### PR DESCRIPTION
**Pull request by ghost**
# ~~**DO NOT MERGE YET!**~~
Gligedit: Ready for review!

~~The code will vary on how many people sign up~~

### CODE REVIEW

**Comment by DeathlyPlays** [comment]
Oh its not done *facepalm*

**Comment by HoeenCoder** [comment]
So what exactly is the change  from normal M&M?

**Comment by ghost** [comment]
More Pokemon are unbanned from mega evolving @HoeenCoder

**Comment by HoeenCoder** [comment]
Cant you use `/tour banlist` for that? not sure might be able to

**Comment by ghost** [comment]
No because the Banlist is ubers, its physically impossible to change what pokemon get banned from mega-evolving without making edits to the specific tier the pokemon pokemon are in in pokedex.js

**Comment by ghost** [comment]
Ok this should be it. Ready for review!

File: mods/sgmnm/formats-data.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	},
+	landorus: {
+		inherit: true,
+		tier: "OU",
+	},
+	marshadow: {
+		inherit: true,
+		tier: "OU",
+	},
+	slaking: {
```
Comment (DeathlyPlays) [inline mods/sgmnm/formats-data.js:32]: Why don't you just add Slaking to the banlist (also would Archeops be broken?)

Comment (ghost) [reply mods/sgmnm/formats-data.js:32]: its UBER meaning it cannot mega evolve, and no, If mix and mega didnt ban archeops than why should i?

Comment (Gligar) [reply mods/sgmnm/formats-data.js]: its UBER meaning it cannot mega evolve, and no, If mix and mega didnt ban archeops than why should i?

Comment (DeathlyPlays) [reply mods/sgmnm/formats-data.js:32]: Mk

**Comment by HoeenCoder** [comment]
This tested?

**Comment by ghost** [comment]
One sec, at school

**Comment by ghost** [comment]
Works

**Pull request merged**